clusterloader2/testing/load/deployment.yaml

```diff
@@ -28,14 +28,10 @@ spec:
             cpu: {{$CpuRequest}}
             memory: {{$MemoryRequest}}
         volumeMounts:
-          {{if (eq (Mod .Index 20) 0 19) }} # .Index % 20 in {0,19} - 10% deployments will have ConfigMap
```

Comment:
So if I understand the change correctly, now every deployment will have both configmap and secret, 90% will be immutable, 10% mutable? If so, can we make it clear in the PR description / commit message?

Reply:
Yes - that's exactly the intention. Updating.

Reply:
done